src/HotChocolate/Core/src/Types.Scalars/WellKnownScalarTypes.cs

```diff
@@ -2,8 +2,9 @@ namespace HotChocolate.Types
 {
     internal static class WellKnownScalarTypes
     {
-        public const string NonEmptyString = nameof(NonEmptyString);
-        public const string PositiveIntType = nameof(PositiveIntType);
-        public const string NegativeInt = nameof(NegativeInt);
+        public const string NonEmptyString = nameof(NonEmptyStringType);
+        public const string PositiveInt = nameof(PositiveIntType);
+        public const string NegativeInt = nameof(NegativeIntType);
+        public const string EmailAddress = nameof(EmailAddressType);
```

Comment:
This was correct before
```suggestion
        public const string NonEmptyString = nameof(NonEmptyString);
        public const string PositiveInt = nameof(PositiveInt);
        public const string NegativeInt = nameof(NegativeInt);
        public const string EmailAddress = nameof(EmailAddress);
```